src/modules/display.js

```diff
@@ -1,14 +1,29 @@
-import { game } from './add.js';
+import id, { apiURL } from './dataAndId.js';
 
-// The function displays the game scores in a list format on the webpage.
-const displayGame = () => {
-  const displayContainer = document.querySelector('.scores');
-  displayContainer.innerHTML = '';
-  game.forEach((singleGame, index) => {
-    const li = document.createElement('li');
-    li.textContent = `${singleGame.userName} : ${singleGame.userScore}`;
-    displayContainer.appendChild(li);
-    li.style.backgroundColor = index % 2 === 0 ? '#f2f2f2' : '#fff';
+/*
+ The function sends a POST request to a specified API endpoint with user and score data and returns
+ the result.
+The user parameter is a string that represents the name or identifier of the user
+ whose score is being submitted.
+ The score parameter is a numerical value representing the score achieved by the
+ user in a game. It is used in the `displayScore` function to send a POST request to the server with
+ the user's score data.
+ The function `displayScore` is returning the `result` property of the JSON response from
+ the API.
+ */
```

Comment:
- [ ] [OPTIONAL] You don't need to describe the functionality of all the functions in your code